### PR TITLE
Reset Schema Script Added

### PR DESCRIPTION
**Pull request by BenBeard115**
Files reorganised and bash script to reset cloud based database added

### CODE REVIEW

File: database/reset_schema.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
+psql -h $DB_IP -U $DB_USERNAME -d $DB_NAME -p 5432 -f schema.sql
```
Comment (shivani-63) [inline database/reset_schema.sh:3]: Nice addition to reset schema

File: database/reset_schema.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+source .env
```
Comment (david-nepo) [inline database/reset_schema.sh:1]: nice bash

**Review by david-nepo — APPROVED** [review]
all looks good - approve!

File: database/schema.sql
Comment (shivani-63) [inline database/schema.sql:1]: I like how you've moved them into a database folder!

**Review by shivani-63 — APPROVED** [review]

**Pull request merged**